vuepress/docs/next/docs/curate/troubleshooting-guide.md

```diff
@@ -3,14 +3,18 @@
 ## How do I access the logs? 
 **A bundle installation or removal has failed. How do I access the logs?**
 
-Currently the Entando Component Manager logs are only available in Kubernetes through Openshift dashboards or CLI tools like K9s and kubectl.
-
+Currently the Entando Component Manager (CM) logs are only available in Kubernetes through Openshift dashboards or CLI tools like K9s and kubectl.
 
 ### Solution
-If your Entando cluster has more than one Entando App custom resource, find the component manager in question using the Entando App name and namespace where the app has been deployed. Assuming you have a single Entando App named `quickstart` and your cluster namespace is called `YOUR-NAMESPACE`, 
-retrieve the component manager logs using this command:
+1. To view the component manager logs, find the CM pod name in your instance:
+```
+ent k get pods
+```
+It will be something like this: `quickstart-cm-deployment-7f74757f97-xnlbn`
+
+2. Using your CM pod name and namespace, use this command to view the logs:
 ```
-kubectl logs -f deployment/quickstart-server-deployment --namespace YOUR-NAMESPACE -c de-container
+ent k logs -f YOUR-PODNAME-7f74757f97-xnlbn -n YOUR-NAMESPACE
```

Comment:
Could add a note that the namespace (-n) is optional if ent has a profile configured.